src/Core/Webservice/ServerRequirementsChecker.php

```diff
@@ -53,27 +54,43 @@
      */
     private $configuration;
 
+    /**
+     * @var HostingInformation
+     */
+    private $hostingInformation;
+
+    /**
+     * @var PhpExtensionCheckerInterface
+     */
+    private $phpExtensionChecker;
+
     /**
      * @param TranslatorInterface $translator
      * @param Configuration $configuration
+     * @param HostingInformation $hostingInformation
+     * @param PhpExtensionCheckerInterface $phpExtensionChecker
      */
-    public function __construct(TranslatorInterface $translator, Configuration $configuration)
-    {
+    public function __construct(
+        TranslatorInterface $translator,
+        Configuration $configuration,
+        HostingInformation $hostingInformation,
```

Comment:
can I add interface for this class?

Reply:
not in the scope of this pull request ;)